.github/workflows/set-default-matrix.yml

```diff
@@ -0,0 +1,127 @@
+name: set-default-matrix
```

Comment:
Why don't you resolve scylla and kafka version in place ?
Have matrix like this:
```
{"kafka_provider": "apache", "provider_version": "8.0", "kafka_connect_mode": "standalone", "scylla_version": "LATEST"}
{"kafka_provider": "apache", "provider_version": "7.9", "kafka_connect_mode": "standalone", "scylla_version": "LATEST"}
...
```